pkg/shp/flags/core_envvvar_value_test.go

```diff
@@ -0,0 +1,35 @@
+package flags
```

Comment:
In the file name, you have three subsequent `v`s where I think only two are desired.

Reply:
Indeed, I renamed in the latest set of changes. Thanks!